pkg/tagger/collectors/workloadmeta_extract.go

```diff
@@ -170,6 +168,11 @@ func (c *WorkloadMetaCollector) processEvents(evBundle workloadmeta.EventBundle)
 func (c *WorkloadMetaCollector) handleContainer(ev workloadmeta.Event) []*TagInfo {
 	container := ev.Entity.(*workloadmeta.Container)
 
+	// Garden containers tagging is specific
+	if container.Runtime == workloadmeta.ContainerRuntimeGarden {
+		return c.handleGardenContainer(container)
+	}
+
```

Comment:
nits: for this kind of `if` condition a switch case make more sens even if we have only one condition.